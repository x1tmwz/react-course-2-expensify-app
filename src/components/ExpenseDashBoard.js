import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilter from './ExpenseListFilter';
import ExpensesSummary from './ExpensesSummary'

const ExpenseDashBoard = () => (
    <div>
        <p>Home Page</p>
        <ExpenseListFilter />
        <ExpensesSummary />
        <ExpenseList />
    </div>
);

export default ExpenseDashBoard;