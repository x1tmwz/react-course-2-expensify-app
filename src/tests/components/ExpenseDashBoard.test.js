import { shallow } from 'enzyme';
import React from 'react';
import ExpenseDashBoard from '../../components/ExpenseDashBoard';


test('should ExpenseDashBorad Will render',()=>{
    const wrapper = shallow(<ExpenseDashBoard />)
    expect(wrapper).toMatchSnapshot();
})
