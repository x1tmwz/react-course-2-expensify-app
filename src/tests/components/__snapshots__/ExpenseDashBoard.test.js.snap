// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should ExpenseDashBorad Will render 1`] = `
<div>
  <p>
    Home Page
  </p>
  <Connect(ExpenseListFilter) />
  <Connect(ExpensesSummary) />
  <Connect(ExpenseList) />
</div>
`;
